tests/components/Geolocation.test.tsx

```diff
@@ -220,6 +220,43 @@ describe('default click handler', () => {
     });
   });
 
+  it('replaces existing location filters with a new location filter in static filters state', async () => {
+    mockAnswersState(mockedStateWithFilters);
+    const actions = spyOnActions();
+    render(<Geolocation />);
+    clickUpdateLocation();
```

Comment:
this should be awaited. I think there are other usages in this file and in LocationBias.test.tsx that need to be updated too

Reply:
done